### PR TITLE
fix: main function bugs

### PR DESCRIPTION
**Pull request by EdwinIZ8**
#### Description of Change

Fixed a grammar error and solved a naming error.

#### Checklist

- [x] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: greedy_algorithms/kruskals_minimum_spanning_tree.cpp [hunk 1]
```diff
@@ -55,15 +55,15 @@ void findMinimumEdge(const int &infinity,
...
 int main() {
-    constexpr int INFINITY = 99999;
-    std::array<std::array<int, 6>, 6> graph{
-        0,        4,        1,        4,        INFINITY, INFINITY,
-        4,        0,        3,        8,        3,        INFINITY,
-        1,        3,        0,        INFINITY, 1,        INFINITY,
-        4,        8,        INFINITY, 0,        5,        7,
-        INFINITY, 3,        1,        5,        0,        INFINITY,
-        INFINITY, INFINITY, INFINITY, 7,        INFINITY, 0};
+    constexpr int POSITIVE_INFINITY = 99999;
```
Comment (realstealthninja) [inline greedy_algorithms/kruskals_minimum_spanning_tree.cpp:58]: why not use `numeric_limits<int>::infinty();` from the cmath library?

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: greedy_algorithms/kruskals_minimum_spanning_tree.cpp (hunk 1 above)
Comment (EdwinIZ8) [reply greedy_algorithms/kruskals_minimum_spanning_tree.cpp:58]: numeric_limits<int>::infinty() returns 0

Comment (realstealthninja) [reply greedy_algorithms/kruskals_minimum_spanning_tree.cpp:58]: Ah I forgot infinity only works for floats. What about numeric_limit<int>::max()?

File: greedy_algorithms/kruskals_minimum_spanning_tree.cpp [hunk 2]
```diff
@@ -55,15 +55,15 @@ void findMinimumEdge(const int &infinity,
...
+    std::array<std::array<int, 6>, 6> graph = {
+        0,        4,        1,        4,        POSITIVE_INFINITY, POSITIVE_INFINITY,
+        4,        0,        3,        8,        3,        POSITIVE_INFINITY,
+        1,        3,        0,        POSITIVE_INFINITY, 1,        POSITIVE_INFINITY,
+        4,        8,        POSITIVE_INFINITY, 0,        5,        7,
+        POSITIVE_INFINITY, 3,        1,        5,        0,        POSITIVE_INFINITY,
+        POSITIVE_INFINITY, POSITIVE_INFINITY, POSITIVE_INFINITY, 7,        POSITIVE_INFINITY, 0};
```
Comment (realstealthninja) [inline greedy_algorithms/kruskals_minimum_spanning_tree.cpp:65]: Also this should be moved to a test function outside main

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: greedy_algorithms/kruskals_minimum_spanning_tree.cpp (hunk 2 above)
Comment (EdwinIZ8) [reply greedy_algorithms/kruskals_minimum_spanning_tree.cpp:65]: Why don't you test main function directly?

**Pull request closed without merging**